doc/tor.1.txt

```diff
@@ -3044,6 +3044,31 @@ The next section describes the per service options that can only be set
    The HAProxy version 1 protocol is described in detail at
    https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt
 
+[[HiddenServiceExportRendPoint]] **HiddenServiceExportRendPoint** **0**|**1**::
+   When HiddenServiceExportCircuitID along with this feature is enabled, export IP
+   and port of the rendezvous point along with the circuit ID. +
+ +
+   When this feature is enabled, tor will export IP and port of the rendezvous
+   point in the following way. +
+ +
+   "PROXY TCP6 fc00::AAAA:BBBB:CCCC:0:ffff:ffff ::1 65535 42\r\n" +
+ +
+      ip = (0xAAAA << 16) + 0xBBBB; +
+      port = 0xCCCC; +
+ +
+   For example, if the IP is 0x7f000001 and the port is 0x1bb, you can translate
+   to 127.30.0.1:443. (Default: 0)
+
+[[HiddenServiceExportInstanceID]] **HiddenServiceExportInstanceID** __N__::
+   When HiddenServiceExportCircuitID is enabled, export an instance ID along
+   with the circuit ID. The maximum allowed value is 65535. +
+ +
+   When this feature is enabled, tor will export an instance ID in the following way. +
+ +
+   "PROXY TCP6 fc00::0:0:0:AAAA:ffff:ffff ::1 65535 42\r\n" +
+ +
+   Where 0xAAAA will be your provided instance ID in hex. (Default: 0)
+
```

Comment:
I can't tell on Github but just make sure that the text paragraph are 79 chars aligned maximum :). If so, great!

Reply:
Fixed in `288823c`.

Before

```
>>> for line in diff.split('\n'):
...     print(f'{"!" if len(line) > 79 else ""}{len(line):02}', line)
... 
79 [[HiddenServiceExportRendPoint]] **HiddenServiceExportRendPoint** **0**|**1**::
!82    When HiddenServiceExportCircuitID along with this feature is enabled, export IP
64    and port of the rendezvous point along with the circuit ID. +
02  +
78    When this feature is enabled, tor will export IP and port of the rendezvous
32    point in the following way. +
02  +
61    "PROXY TCP6 fc00::AAAA:BBBB:CCCC:0:ffff:ffff ::1 65535 42
03 " +
02  +
37       ip = (0xAAAA << 16) + 0xBBBB; +
22       port = 0xCCCC; +
02  +
!80    For example, if the IP is 0x7f000001 and the port is 0x1bb, you can translate
34    to 127.30.0.1:443. (Default: 0)
00 
75 [[HiddenServiceExportInstanceID]] **HiddenServiceExportInstanceID** __N__::
76    When HiddenServiceExportCircuitID is enabled, export an instance ID along
61    with the circuit ID. The maximum allowed value is 65535. +
02  +
!87    When this feature is enabled, tor will export an instance ID in the following way. +
02  +
55    "PROXY TCP6 fc00::0:0:0:AAAA:ffff:ffff ::1 65535 42
03 " +
02  +
70    Where 0xAAAA will be your provided instance ID in hex. (Default: 0)
00 

```
After

```
>>> for line in diff.split('\n'):
...     print(f'{"!" if len(line) > 79 else ""}{len(line):02}', line)
... 
79 [[HiddenServiceExportRendPoint]] **HiddenServiceExportRendPoint** **0**|**1**::
72    When HiddenServiceExportCircuitID along with this feature is enabled,
74    export IP and port of the rendezvous point along with the circuit ID. +
02  +
78    When this feature is enabled, tor will export IP and port of the rendezvous
32    point in the following way. +
02  +
61    "PROXY TCP6 fc00::AAAA:BBBB:CCCC:0:ffff:ffff ::1 65535 42
03 " +
02  +
37       ip = (0xAAAA << 16) + 0xBBBB; +
22       port = 0xCCCC; +
02  +
70    For example, if the IP is 0x7f000001 and the port is 0x1bb, you can
44    translate to 127.30.0.1:443. (Default: 0)
00 
75 [[HiddenServiceExportInstanceID]] **HiddenServiceExportInstanceID** __N__::
76    When HiddenServiceExportCircuitID is enabled, export an instance ID along
61    with the circuit ID. The maximum allowed value is 65535. +
02  +
63    When this feature is enabled, tor will export an instance ID
26    in the following way. +
02  +
55    "PROXY TCP6 fc00::0:0:0:AAAA:ffff:ffff ::1 65535 42
03 " +
02  +
70    Where 0xAAAA will be your provided instance ID in hex. (Default: 0)
```